adapter/src/components/__tests__/Alerts.test.js

```diff
@@ -49,9 +49,10 @@ describe('Alerts', () => {
 
     it('can hide an alert', async () => {
         const msg = 'TEST ALERT'
+        const options = { permanent: true }
```

Comment:
This ensures the alert does't auto-hide. We need this to verify that hiding programmatically does indeed clean up the DOM.